### PR TITLE
Documentation for query-time biases influence

### PR DESCRIPTION
**Pull request by jmbtutor**

### CODE REVIEW

**Comment by jmbtutor** [comment]
@willwarren This is the documentation for query-time biasing influence. Please review.

**Comment by jmbtutor** [comment]
@osman Please also review.

File: src/main/java/com/groupbyinc/api/Query.java [hunk 1]
```diff
@@ -1215,6 +1220,7 @@ public Query addQueryUrlParams(String key, String value) {
      *     { "biasing": {
      *         "bringToTop": ["productId1","productId3","productId2"]
      *          "augmentBiases": false,
+     *          "biasInfluence": 5.0,
```
Comment (codeallthethingz) [inline src/main/java/com/groupbyinc/api/Query.java]: biasInfluence --> influence

File: src/main/java/com/groupbyinc/api/Query.java [hunk 2]
```diff
@@ -1203,11 +1203,16 @@ public Query addQueryUrlParams(String key, String value) {
      * <code>
      * Add a biasing profile, which is defined at query time. Possible settings
      * include:
+     *
      *  - `bringToTop`: A list of product IDs to bring to the top of the result set. This list
      *  will ensure that the products are included in the result set and appear in the order
      *  defined.
-     *  - A list of biases, which either override or augment biasing profiles defined
-     *  in Command Center. By default, biases defined here will override the ones defined
+     *  - `augmentBiases`: If true, the biases defined here will augment biasing profiles defined in Command Center.
```
Comment (willwarren) [inline src/main/java/com/groupbyinc/api/Query.java]: augmentBiases --> augment

**Comment by jmbtutor** [comment]
@willwarren Does the specified influence only apply to the query-time biases or does it also override the bias profile influence set in Command Center?

**Comment by jmbtutor** [comment]
@willwarren Clarified wording for query-time influence.

**Comment by codeallthethingz** [comment]
https://github.com/groupby/api-java/blob/GB-1971/src/main/java/com/groupbyinc/api/Query.java#L1259 needs to be indented one more space.

@jmbtutor small change that wasn't your fault.

**Pull request merged**